etl/stock_hard_delete_pattern.py

```diff
@@ -0,0 +1,64 @@
+from sqlalchemy import create_engine, text
+from datetime import datetime
+from io import StringIO
+import os
+import boto3
+import pandas as pd
+
+
+# Connect MySQL db
+db_user = "admin"
+db_password = "password"
+db_host = "my-db-instance.cxjkxlbvg4uh.ap-northeast-1.rds.amazonaws.com"
+db_name = "stock_db"
```

Comment:
Can you delete this. All these details will be in the `DB_URL` environment variable